### PR TITLE
Percentage ties

### PR DESCRIPTION
**Pull request by araspaas**
created methods and tests for find tied games and percentage tied. passed tests for both. 
@sciencefixion @rrabinovitch @gabichuelas

### CODE REVIEW

**Review by sciencefixion — APPROVED** [review]
Looks good! I might delete my large game teams fixture and just use yours.

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -54,6 +54,17 @@ def percentage_visitor_wins
...
   end
 
+  def find_tied_games
+    games.find_all do |game|
+      game.away_goals == game.home_goals
+    end
+  end
+
+  def percentage_ties
+    percentage = find_tied_games.count.fdiv(games.count) * 100
```
Comment (rrabinovitch) [inline lib/stat_tracker.rb]: @gabichuelas so based on your note in slack, it sounds like this should just be `percentage = find_tied_games.count.fdiv(games.count)` without the `* 100`, yes?

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -72,6 +72,30 @@ def test_away_wins_percentage
...
+  end
+
+  def test_percentage_ties
+    locations = {
+      games: './fixtures/game_statistics_games_fixture.csv',
+      teams: './fixtures/teams_fixture.csv',
+      game_teams: './fixtures/game_teams_game_statistics_fixture.csv'
+    }
+    stat_tracker = StatTracker.from_csv(locations)
+    assert_equal 4.0, stat_tracker.percentage_ties
```
Comment (rrabinovitch) [inline test/stat_tracker_test.rb]: @gabichuelas and this `0.04`?

**Review by rrabinovitch — COMMENTED** [review]
Looks good, Austin! We should just standardize what we're calling our expanded fixture files - we can talk about this when we meet later today. Also take a look at Gaby's message about the spec harness in slack - there are some small details we might need to change in your calculations (per my line specific comments on the changed files). Let's get confirmation/clarity on that first before merging.

File: lib/stat_tracker.rb (hunk 1 above)
Comment (gabichuelas) [reply lib/stat_tracker.rb]: @rrabinovitch yeah, and then we have to changed related tests to account for the change.

File: test/stat_tracker_test.rb (hunk 2 above)
Comment (gabichuelas) [reply test/stat_tracker_test.rb]: yup

**Review by gabichuelas — CHANGES_REQUESTED** [review]
@evilaspaas1 just gotta make those changes regarding percentages, and also will probably need to change some other stuff around because we need to make the string to integer conversion for things like goals and whatever else is a number, somewhere in the methods and not in the initialize method for each object. We can talk more at meeting

**Review by rrabinovitch — APPROVED** [review]

**Pull request merged**